packages/record/src/record/record-login-flow.ts

```diff
@@ -148,12 +153,11 @@ export const recordLoginFlowSession = async ({
         });
 
         // Navigate to the final login flow url and flush any pending payloads
+        logger.debug("Reloading page to capture application storage...");
         await loginDataPage.goto(finalLoginUrl, {
-          waitUntil: "domcontentloaded",
-        });
-        await loginDataPage.evaluate(() => {
-          (window as ModifiedWindow).__meticulous?.flushPendingPayloads?.();
+          waitUntil: "load",
```

Comment:
I thought I hit a case on one website where changing this to "load" caused it to hang indefinitely... but I can't reproduce it, so going to assume it's safe for now and if we notice it hanging we can fix it.